### ISSUE TITLE
Admin Email to everybody on server

### ISSUE THREAD

**Issue by dsp1986** [friendica/friendica#773]
As admin of a server I would like to be able to send private messages also to people who are not connected with my account, but use my server. It would help me to take care of the community on my server.
## Background:
1. Some friends have difficulties finding their way around in the beginning. Some have had trouble connecting. It would be nice to be able to send them a welcome message and show them round the site a bit.
2. In order to make it as easy as possible for friends to sign up to my server I like to have registration policy open. (I have people from very different timezones on my server and don't want them to wait until I see their registration and approve it.)
   Now I have a registration with an email I don't know, inactive for 4 days now (no login) and according to cleantalk the email has spam activity on 111 web sites. 
   In this case (and possible future ones which might be less clear) I would like to be able to contact this "person" as admin through friendica and ask who he is.

[thread on Friendica Developers](https://friendika.openmindspace.org/display/friendicadevelopers/1325444)

**Comment by tobiasd** [comment]
For the 1st point, there is now an [addon](https://github.com/friendica/friendica-addons/tree/master/notifyall) in the addon repository.

**Comment by MrPetovan** [comment]
> For the 1st point, there is now an addon in the addon repository.

This isn't exactly the same. Like @realkinetix, I assume OP wanted to contact a specific person regardless of their private message policy.

The second point is about contacting people by email directly from Friendica (not a private message).

**Comment by realkinetix** [comment]
Correct.  For me, bulk messaging everyone is not the goal.  Being able to send any user on my node a private message is.

**Comment by tobiasd** [comment]
Which is exactly what this feature request is about, isn't it? *I would like to be able to send private messages also to people who are not connected with my account*

**Comment by MrPetovan** [comment]
My comment was only about your reply mentioning the `notifyall` addon.
